pkg/helpers/helpers.go

```diff
@@ -334,10 +335,16 @@ func ApplyDirectly(
 			result.Result, result.Changed, result.Error = ApplyMutatingWebhookConfiguration(
 				client.AdmissionregistrationV1(), t)
 		case *apiregistrationv1.APIService:
+			t.ObjectMeta.Annotations = make(map[string]string)
+			checksum := md5.Sum(t.Spec.CABundle)
+			t.ObjectMeta.Annotations["caBundle-checksum"] = string(checksum[:]) // to trigger the update when caBundle changed
```

Comment:
I think we commonly use sha256 like this

```
fmt.Sprintf("%x", sha256.Sum256(data))
```

Reply:
Yes, changed to use sha56.